fortnitepy/party.py

```diff
@@ -1099,6 +1103,9 @@ def __init__(self, party: 'PartyBase',
                     'tournamentId': '',
                     'eventWindowId': '',
                     'regionId': 'EU',
+                    'linkId': {
+                        'mnemonic': 'playlist_defaultDuo',
```

Comment:
Why is this here? Doesn't seem like this is needed in any way?

Reply:
its for fix the party visual when the game mode is selected bc it will they on the basic one..